<launch>
    <arg name="dsrc_address" default="192.168.88.40" />
    <arg name="dsrc_listening_port" default="1516" />
    <arg name="listening_port" default="5398" />
    <arg name="wave_cfg_file" default="$(find dsrc_driver)/etc/wave.json" />
    <arg name="remap_ns" default="/"/>

    <group ns="$(arg remap_ns)">
        <node name="dsrc" pkg="dsrc_driver" type="dsrc_driver_node" output="screen">
            <param name="dsrc_address" value="$(arg dsrc_address)" />
            <param name="dsrc_listening_port" value="$(arg dsrc_listening_port)" />
            <param name="listening_port" value="$(arg listening_port)" />
            <param name="wave_cfg_file" value="$(arg wave_cfg_file)" />
        </node>
    </group>
</launch>
